README.md

```diff
@@ -71,10 +71,20 @@ func main() {
 }
 ```
 
-## Building From Source
+## Contributing
 
-Run benchmarks
+1. [Install Go](https://go.dev/doc/install) 1.19+
```

Comment:
🤦‍♂️ this is incorrect after I created https://github.com/celestiaorg/rsmt2d/pull/211